pyprep/utils.py

```diff
@@ -150,3 +151,39 @@ def make_random_mne_object(
     info = mne.create_info(ch_names=ch_names, sfreq=sfreq, ch_types=ch_types)
     raw = mne.io.RawArray(signal, info)
     return raw, n_freq_comps, freq_range
+
+
+def verify_free_ram(data, n_samples, n_channels, max_prop=0.95):
+    """Check if enough memory is free to run ransac with the given parameters.
+
+    Parameters
+    ----------
+    data : ndarray
+        2-D EEG data
+    n_samples : int
+        Number of samples to use for computation of ransac.
+    n_channels : int
+        Number of channels to process per chunk.
+    max_prop: float
```

Comment:
```suggestion
    max_prop : float
```

I wonder why [flake8-docstrings](https://pypi.org/project/flake8-docstrings/) does not catch this

Reply:
Yeah, weird... I keep forgetting the spaces on these because I've only ever used Google-style Python docstrings before, gotta fight some well-learned reflexes to conform to numpydoc!